share/doc/wake/tutorial.md

```diff
@@ -1055,7 +1055,7 @@ In `buildHeaders`, we've used the `sources` command to find all the header
 files in the same directory and pass them as legal inputs to gcc -- the
 keyword `here` expands to the directory of the `.wake` file.  The second
 argument to `sources` is a regular expression to select which files to
-return. We've used ``` `` ```s here which define regular expression literals
+return. We've used ``` `` ```s (backticks) here which define regular expression literals
```

Comment:
This would be the very simple way to clarify the markdown mess -- it looked good in the rendered view, at least...